src/Api/Billing/Public/Models/OrganizationSubscriptionUpdateRequestModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Bit.Api.Billing.Public.Models;
+
+public class OrganizationSubscriptionUpdateRequestModel
```

Comment:
Looks like the ticket also calls for the ability to update storage in the description. Might want to check with the reporter on whether that is actually required.